sky/backends/cloud_vm_ray_backend.py

```diff
@@ -3701,6 +3704,20 @@ def _check_existing_cluster(
                       if resources.cloud is not None else clouds.Cloud)
         task_cloud.check_cluster_name_is_valid(cluster_name)
 
+        if to_provision is None:
+            logger.info(
+                f'The cluster {cluster_name!r} was autodowned or manually '
+                'terminated on the cloud console. Using the original resources '
+                'to provision a new cluster.')
```

Comment:
I think the meaning of "original" here is a bit unclear. Can we change this like following?
```suggestion
                'terminated on the cloud console. Using the same resources as the previously terminated one'
                'to provision a new cluster.')
```